frontend/src/components/Map/SearchMap/MoveHandler.tsx

```diff
@@ -10,17 +10,20 @@ interface Props {
 const MoveHandler: React.FC<Props> = ({ onMove }) => {
   const map = useMap();
 
-  const handleMove = useCallback(
-    debounce(
-      () => {
-        onMove(map.getBounds());
-      },
-      1000,
-      false,
-    ),
+  const debouncedMove = useMemo(
+    () =>
+      debounce(
+        () => {
+          onMove(map.getBounds());
+        },
+        1000,
+        false,
+      ),
     [map, onMove],
   );
 
+  const handleMove = useCallback(debouncedMove, [debouncedMove]);
```

Comment:
As I think _(I could be wrong)_ `useMemo(() => func, deps)` is the same as `useCallback(func, deps)` so:
```js
const handleMove = useCallback(useMemo(() => debounce(), depsA), depsB);
```
could be simply as:
```js
const handleMove = useMemo(() => debounce(), depsA);
```
and as `debounce()` is also returning another function, maybe:
```js
const handleMove = useMemo(debounce(), deps);
```

Reply:
Yes good point, thx !